molecule/default/prepare.yml

```
@@ -13,9 +14,9 @@
   tasks:
     - name: Group hosts by OS distribution
       group_by:
-        key: os_{{ ansible_distribution }}_{{ ansible_distribution_version }}
-- name: Wait for systemd to complete initialization (Fedora 34)
-  hosts: os_Fedora_34
+        key: os_{{ ansible_distribution }}
+- name: Wait for systemd to complete initialization (Fedora)
+  hosts: os_Fedora
   tasks:
     - name: Wait for systemd to complete initialization # noqa 303
       command: systemctl is-system-running
```

I think this was missed previously. I'm not sure if you wanted to sort the other settings for this task as well but it's outside the scope of what I can suggest.

```suggestion
      ansible.builtin.command: systemctl is-system-running
```